docs/pages/index.mdx

```diff
@@ -1,29 +1,37 @@
 # Introduction
 
-Mod is a standard for creating Mini-apps. Mod currently only supports the Farcaster protocol, and Mini-apps can be one of two kinds: Mini-apps for creating content, or Mini-apps for displaying embeds.
-Mod Mini-apps are currently defined via a JSON configuration file, and are reviewed and approved into the registry on a case by case basis to ensure security, performance and quality.
+Mod is a standard for creating Embedded Modules. Mod currently only supports the Farcaster protocol, and Mods can be one of two kinds: Mods for creating content, or Mods for displaying rich embeds.
```

Comment:
i'd argue embedded modules is less clear than miniapps but i suppose they are both niche words that people don't typically know so up to you

regardless of that, i think think the 'apps that live inside of other apps' definition is quite useful for new people trying to understand what mods are so might be worth considering adding that in this definition

Reply:
Mod is a standard for creating apps that live inside of other apps. We call this a Module, or Mod in short

Reply:
Love it